docs/niscope/class.rst

```diff
@@ -1497,6 +1520,106 @@ get_equalization_filter_coefficients
                 niscope.Session repeated capabilities container, and calling this method on the result.
 
 
+get_ext_cal_last_date_and_time
+------------------------------
+
+    .. py:currentmodule:: niscope.Session
+
+    .. py:method:: get_ext_cal_last_date_and_time()
+
+            Returns the date and time of the last external calibration performed.
+
+            
+
+            .. note:: The NI 4050 and NI 4060 are not supported.
+
+
+
+            :rtype: int
+            :return:
+
+
+                    Indicates the **month** of the last calibration.
+
+                    
+
+
+
+get_ext_cal_last_temp
+---------------------
+
+    .. py:currentmodule:: niscope.Session
+
+    .. py:method:: get_ext_cal_last_temp()
+
+            Returns the **Temperature** during the last external calibration procedure.
```

Comment:
in Celsius?

Reply:
Yes! Now clarified in the documentation.